docs/start/getting-started/fragments/angular/setup.md

```diff
@@ -52,8 +69,25 @@ When you initialize a new Amplify project, a few things happen:
 Inside the app directory, install the Amplify Angular library and run your app:
 
 ```bash
-npm install aws-amplify @aws-amplify/ui-angular
+npm install --save @aws-amplify/auth @aws-amplify/api @aws-amplify/pubsub @aws-amplify/storage @aws-amplify/ui-angular
 ng serve
 ```
 
 The `@aws-amplify/ui-angular` package is a set of Angular components and an Angular provider which helps integrate your application with the AWS-Amplify library.  It supports Angular 5.0 and above.  It also includes a [supplemental module](#ionic-4-components) for Ionic-specific components.
+
+<amplify-callout>Angular CLI output warnings: if you see CommonJS or AMD dependencies optimization bailouts warnings using Angular +9 you can use this [gist](https://gist.github.com/gsans/8982c126c4fef668c094ff288f04241b) to remove them. More details about these [here](https://angular.io/guide/build#configuring-commonjs-dependencies).
```

Comment:
Instead of speaking in generalities, are there salient excerpts we can extract from the error messages?

e.g.,

"If you get an `TechnicalReferenceFromOutput` warning while using Angular 9+, \[see these notes\](http://notes.link)".

Reply:
There's no specific error. Output is actually worst: 
```
WARNING in /Users/ehanchett/programming/work/eig/front/crty-ng-ui/node_modules/@aws-sdk/client-s3/dist/es/runtimeConfig.browser.js depends on @aws-sdk/eventstream-serde-browser. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```